src/DynamoCoreWpf/Controls/InstalledPackagesControl.xaml

```diff
@@ -0,0 +1,300 @@
+                                                                        AncestorType={x:Type ItemsPresenter}}}">
+                        <Expander Name="PackageHeader"
+                                          IsExpanded="{Binding Path=Model.TypesVisibleInManager}"
+                                          Style="{StaticResource InstalledPackagesExpanderStyle}">
+                            <Expander.Header>
+                                <Grid x:Name="PackageGrid"  Width="400">
+                                    <DockPanel HorizontalAlignment="Stretch"
+                                                   >
+                                        <StackPanel HorizontalAlignment="Left"
+                                                        Orientation="Horizontal"
+                                                        Margin="5,5,5,7">
+                                            <TextBlock Text="{Binding Path=Model.Name}"
+                                                           ToolTip="{Binding Path=Model.Name}"
+                                                           FontSize="11"
+                                                           Width="150"
+                                                           TextTrimming="CharacterEllipsis"
+                                                           Margin="0,0,10,0"
+                                                           Foreground="#E5E5E5"></TextBlock>
+                                            <TextBlock Text="{Binding Path=Model.VersionName}"
+                                                           MinWidth="50"
+                                                           FontSize="10"
+                                                           Foreground="#E5E5E5"></TextBlock>
+                                            <TextBlock Text="{x:Static p:Resources.InstalledPackageViewPendingInstallButton}"
+                                                           Visibility="{Binding Path=Model.EnableOldMarkedForUnistallState, Converter={StaticResource BooleanToVisibilityCollapsedConverter}}"
+                                                           FontSize="11"
+                                                           FontStyle="Italic"
+                                                           Foreground="Red"></TextBlock>
+                                        </StackPanel>
+                                        <StackPanel HorizontalAlignment="Right" Orientation="Horizontal" Width="Auto">
+                                            <Button Name="MoreButton"
+                                                            Click="MoreButton_OnClick"
+                                                            Style="{StaticResource FlatButtonStyle}">
+                                                <Image Source="/DynamoCoreWpf;component/UI/Images/More.png"/>
```

Comment:
this button looks kind of blurry, is it larger than the .png?

Reply:
Thanks, Yes, it was larger.